static/calculate_score.html

```diff
@@ -196,16 +196,16 @@ <h5>3. Select from Filtered Studies&nbsp;</h5>
                     <span title="A unique study identifier. In our database, we use GWAS Catalog study identifiers. As long as this is unique for each study, it can be whatever you want.">Study&nbsp;ID<sup>?</sup></span>, 
                     <span title="The Experimental Factor Ontology (EFO) trait the GWAS deals with.">Trait<sup>?</sup></span>, 
-                    <span title="The dbSNP reference (rs) number of the SNP.">Rsid<sup>?</sup></span>, 
+                    <span title="The dbSNP reference (rs) number of the SNP.">RsID<sup>?</sup></span>, 
                     <span title="The chromosome the SNP resides on.">Chromosome<sup>?</sup></span>, 
                     <span title="The position of the SNP in the reference genome.">Position<sup>?</sup></span>, 
-                    <span title="The allele that confers Risk or Protection.">Risk&nbsp;Allele<sup>?</sup></span>, 
-                    <span title="Computed in the GWAS study, a numerical value of the odds that those in the case group have the allele of interest over the odds that those in the control group have the allele of interest.">Odds&nbsp;Ratio<sup>?</sup></span>, and 
-                    <span title="The probability that the risk allele confers the amount of risk stated.">P-value<sup>?</sup></span>. 
+                    <span title="The allele that confers risk or protection.">Risk&nbsp;Allele<sup>?</sup></span>, 
+                    <span title="Computed in the GWA study, a numerical value representing the odds that those in the case group carry the allele of interest over the odds that those in the control group carry the allele of interest.">Odds&nbsp;Ratio<sup>?</sup></span>, and 
+                    <span title="The probability that the risk associated with the allele of interest is observed by chance.">P-value<sup>?</sup></span>. 
                     Optional column headers that will be included if present are:
                     <span title="The citation information for the study.">Citation<sup>?</sup></span>
-                    and <span title ="Trait description for this study in the authors own words.">Reported&nbsp;Trait<sup>?</sup></span>.
-                    All other columns in the file will be ignored. If more than one odds ratio exists for a dbSNP reference (rs) number, the odd ratio and corresponding risk allele with the most significant p-value will be used.</p>
+                    and <span title ="Trait description for this study in the author's own words.">Reported&nbsp;Trait<sup>?</sup></span>.
+                    All other columns in the file will be ignored. If more than one odds ratio exists for an rsID, the odds ratio and corresponding risk allele with the most significant p-value will be used.</p>
```

Comment:
Leave as a dbSNP reference (rs) number. I think Justin would like more reference to the rsID as a dbSNP reference (rs) number.